HospitalWebApp/Controllers/PatientController.cs

```diff
@@ -1,12 +1,17 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
 using HealthcareBase.Model.Users.Patient;
+using HealthcareBase.Model.Users.UserAccounts;
+using HealthcareBase.Model.Users.UserAccounts.Registration;
 using HealthcareBase.Service.UsersService.PatientService;
-using HospitalWebApp.Adapters;
+using HospitalWebApp.Mappers;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Hosting;
+using PatientRegistrationService = HealthcareBase.Service.UsersService.RegistrationService.PatientRegistrationService;
```

Comment:
What represents this assignment?

Reply:
Back then, I haven't noticed that I have two classes with same name, this assignment specifies which class I want to use.